src/App.css

```diff
@@ -47,9 +106,9 @@ div {
   background-color: orange;
 }
 
-div:hover {
+/* div:hover {
   cursor: pointer;
-}
+} */
```

Comment:
- [ ]Good job so far, however, please do not leave commented pieces of code in your project,  Commented code adds unnecessary clutter to your project files, making it harder to read and understand the active, functioning code. This can lead to confusion for you and other developers who work on the project. please remove it. :slightly_smiling_face: